AUTHORS

```diff
@@ -0,0 +1,24 @@
+Chip Senkbeil <chip.senkbeil@gmail.com>
```

Comment:
I would like to name this file as `CONTRIBUTORS.md`. We can use [this example](https://github.com/jonathantneal/svg4everybody/blob/master/CONTRIBUTORS.md) as model, but we should not list every change made in this file, just list the people.

Reply:
Also, I prefer to list the contributors and only link to their GitHub profile. By listing their emails we create a free spam list. :P

Reply:
I'm ok with using `CONTRIBUTORS` as opposed to `AUTHORS`. But at the same time I don't see a point why not show their emails. That's a personal choice everyone makes, to make their emails private or not. If I set it to be public (which you can clearly and easily find it through git history), there's no reason to not show it here in this file too.

Reply:
People may not what to add their email to this file, so I prefer to just make a list and link to their GitHub profile.

Reply:
All right, I'm gonna change it to GH handles.